src/components/SharedComponents/UserText.js

```diff
@@ -0,0 +1,134 @@
+import TableRenderer, { tableModel } from "@native-html/table-plugin";
+import linkifyHtml from "linkify-html";
+import MarkdownIt from "markdown-it";
+import * as React from "react";
+import {
+  ScrollView, useWindowDimensions
+} from "react-native";
+import HTML from "react-native-render-html";
+import WebView from "react-native-webview";
+import sanitizeHtml from "sanitize-html";
+
+const ALLOWED_TAGS = ( `
+  a
+  abbr
+  acronym
+  b
+  blockquote
+  br
+  cite
+  code
+  del
+  div
+  dl
+  dt
+  em
+  h1
+  h2
+  h3
+  h4
+  h5
+  h6
+  hr
+  i
+  img
+  ins
+  li
+  ol
+  p
+  pre
+  s
+  small
+  strike
+  strong
+  sub
+  sup
+  table
+  tbody
+  td
+  t
+  th
+  thead
+  tr
+  tt
+  ul
+` ).split( /\s+/m ).filter( e => e !== "" );
+
+const ALLOWED_ATTRIBUTES_NAMES = (
+  "href src width height alt cite title class name abbr value align target rel"
+  // + "xml:lang style controls preload"
```

Comment:
Why are these commented out?

Reply:
I took these constants from the web component and they were commented out there. I didn't know why so I just kept it. Should I remove them?

Reply:
Yes please.